Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsViewModel.kt

```diff
@@ -107,19 +108,23 @@ class PersonDetailsViewModel @AssistedInject constructor(
             personCertificates.certificates.find { it is VaccinationCertificate }?.let { certificate ->
                 val vaccinatedPerson = vaccinatedPerson(certificate)
                 if (vaccinatedPerson != null) {
-                    val daysUntilImmunity = vaccinatedPerson.getDaysUntilImmunity()
-                    val vaccinationStatus = vaccinatedPerson.getVaccinationStatus()
-                    val daysSinceLastVaccination = vaccinatedPerson.getDaysSinceLastVaccination()
-                    val boosterRule = vaccinatedPerson.boosterRule
-                    add(
-                        VaccinationInfoCard.Item(
-                            vaccinationStatus = vaccinationStatus,
-                            daysUntilImmunity = daysUntilImmunity,
-                            boosterRule = boosterRule,
-                            daysSinceLastVaccination = daysSinceLastVaccination,
-                            hasBoosterNotification = vaccinatedPerson.hasBoosterNotification
+                    try {
+                        val daysUntilImmunity = vaccinatedPerson.getDaysUntilImmunity()
+                        val vaccinationStatus = vaccinatedPerson.getVaccinationStatus()
+                        val daysSinceLastVaccination = vaccinatedPerson.getDaysSinceLastVaccination()
+                        val boosterRule = vaccinatedPerson.boosterRule
+                        add(
+                            VaccinationInfoCard.Item(
+                                vaccinationStatus = vaccinationStatus,
+                                daysUntilImmunity = daysUntilImmunity,
+                                boosterRule = boosterRule,
+                                daysSinceLastVaccination = daysSinceLastVaccination,
+                                hasBoosterNotification = vaccinatedPerson.hasBoosterNotification
+                            )
                         )
-                    )
+                    } catch (e: Exception) {
+                        Timber.e(e, "creating VaccinationInfoCard.Item failed")
```

Comment:
what happens in UI in this case ?

Reply:
`VaccinationInfoCard` is not visible in UI. But if the trigger for this crash is the same as for the bug mentioned in the PR description then this should happen only when the user is deleting certificates so he/she will be forwarded to a list of persons immediately and this shouldn't be a concern.